colcon-build-and-test/action.yaml

```diff
@@ -46,12 +46,6 @@ runs:
         DEBIAN_FRONTEND=noninteractive rosdep install -yqq --from-paths . --ignore-src --rosdistro ${{ inputs.rosdistro }}
       shell: bash
 
-    - name: Set up colcon-mixin
-      run: |
-        colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
-        colcon mixin update default
-      shell: bash
-
```

Comment:
I think we need to keep this since not all Docker images have set up colcon-mixin.
Instead of removing this, we can check if `default` already exists before adding it.

Reply:
Thanks. Changed to ignore the error instead of removing the colcon mixin step.